main.py

```diff
@@ -19,7 +20,11 @@ def main():
     """
     signal = Signals("11011100111")
     signal.display()
-    unipolar_nrz_encode(signal)
+    # unipolar_nrz_encode(signal)
+    # signal.display()
+    encoded_bits = polar_nrz_encode(signal)
```

Comment:
```suggestion
    polar_nrz_encode(signal)
```
The return value of the function is None.